.github/actions/code-pushup/src/runner.ts

```diff
@@ -134,9 +135,32 @@ async function run(): Promise<void> {
       logger.setVerbose(true);
     }
 
-    const options: Options = {
-      bin: 'npx nx code-pushup --nx-bail --',
-    };
+    const isMonorepo = core.getBooleanInput('monorepo');
+
+    const options: Options = isMonorepo
+      ? {
+          jobId: 'monorepo-mode',
+          monorepo: 'nx',
+          nxProjectsFilter: '--with-target=code-pushup --exclude=workspace',
```

Comment:
Users could have custom target names. Or Nx-Plugin for example supports this with better DX over the [plugin options](https://github.com/code-pushup/cli/blob/main/packages/nx-plugin/src/plugin/README.md#options). Would it be possible to provide this an option with a default value?

Reply:
We already support custom target names via the [`task` option](https://github.com/code-pushup/cli/tree/main/packages/ci#options:~:text=task,project%20in%20monorepo%20mode). But that's not related to this piece of code.

Unlike `@code-pushup/ci`, the local action isn't user-facing. Its sole purpose is to run `code-pushup` on _this and only this repo_ from source. So, for simplicity, we provide the options directly. For the full context, see #1093.